include/swift/SIL/SILInstruction.h

```diff
@@ -7978,6 +7982,131 @@ inline DestructureTupleInst *DestructureTupleResult::getParent() {
   return cast<DestructureTupleInst>(Parent);
 }
 
+/// SWIFT_ENABLE_TENSORFLOW
+/// A result for the graph_op instruction. See documentation for
+/// graph_op for more information.
+class GraphOperationResult final : public MultipleValueInstructionResult {
+public:
+  GraphOperationResult(unsigned Index, SILType Type,
+                       ValueOwnershipKind OwnershipKind)
+  : MultipleValueInstructionResult(ValueKind::GraphOperationResult, Index,
+                                   Type, OwnershipKind) {}
+
+  static bool classof(const SILNode *N) {
+    return N->getKind() == SILNodeKind::GraphOperationResult;
+  }
+
+  GraphOperationInst *getParent();
```

Comment:
Thanks for the catch, I inlined the function.
I didn't inline it originally because I copied the code from [`DestructureTupleInst`](https://github.com/dan-zheng/swift/blob/534810d232e6c2337bc824b6745f53f14c2385bb/include/swift/SIL/SILInstruction.h#L7980).